velox/exec/tests/NestedLoopJoinTest.cpp

```diff
@@ -255,6 +255,44 @@ TEST_F(NestedLoopJoinTest, basicCrossJoin) {
       "SELECT * FROM t, (SELECT * FROM UNNEST (ARRAY[10, 17, 10, 17, 10, 17, 10, 17])) u");
 }
 
+TEST_F(NestedLoopJoinTest, testOuterJoin) {
```

Comment:
Maybe rename this testOuterJoinWithoutCondition so that the inner lambda can be renamed testOuterJoin ?

Reply:
FYI: per naming convention, test methods should not have 'test' prefix

Reply:
@mbasmanova : Yes, thanks my bad.